src/renderer/atlas/shader_ps.hlsl

```diff
@@ -66,18 +72,24 @@ float4 main(float4 pos: SV_Position): SV_Target
     // Colored cursors are drawn "in between" the background color and the text of a cell.
     if ((cell.flags & 2) && cursorColor != 0xffffffff)
     {
+        // The cursor texture is stored at the top-left-most glyph cell.
+        // Cursor pixels are either entirely transparent or opaque.
+        // --> We can just use .a as a mask to flip cursor pixels on or off.
         color = alphaBlendPremultiplied(color, decodeRGBA(cursorColor) * glyphs[cellPos].a);
     }
 
     // Layer 2:
     // The cell's glyph potentially drawn in the foreground color
     {
         float4 glyph = glyphs[decodeU16x2(cell.glyphPos) + cellPos];
+        // TODO: Can we convert to linear space right during DirectWrite rasterization?
+        // Pros: Saves us cycles in this shader. Cons: Precision loss?

```

Comment:
How expensive is it to do this for every pixel?

Reply:
Probably not much so I removed the comment.